### PR TITLE
Create project

### PR DESCRIPTION
**Pull request by LinKassem**
@silvermist @MariamMohamedFawzy @mansoursaid001 Can you please review.

to test
-first drop the database and re-create it
-then run "php artisan migrate:refresh --seed" (I am sorry for not creating a migration, I could not get the migration to work.)
-login as an admin and
-then try to add, edit, delete a project

Thank you!

### CODE REVIEW

**Comment by LinKassem** [comment]
Once you merge can you please add this comment to the web group on fb
"Guys the projects table on the development branch is now updated..so please drop the database and recreate it and then run the migrations and the seeds via 'php artisan migrate:refresh --seed'. I am really sorry for this but I have spent nearly 1:30 hrs trying to do it in a separate migration but did not succeed! Lin :D"

File: app/Http/Controllers/ProjectsController.php [hunk 1]
```diff
@@ -22,31 +28,77 @@ public function show($id){
...
-        $project->save();
-        return $project;
+
+        // validations
+        $this->validate($request, [
+            'name' => 'required|min:3|max:12',
+            'description' => 'required|min:3s',
+            'created_by' => 'required'
+        ]);
+        error_log($request->created_by);
```
Comment (mansoursaid001) [inline app/Http/Controllers/ProjectsController.php]: This should be removed

**Comment by LinKassem** [comment]
@MariamMohamedFawzy can you please recheck? Thank you!

**Comment by marmohamed** [comment]
@LinKassem in /projects/{id}/edit
could u fill the fields with the values of the currently edited project?

**Comment by LinKassem** [comment]
@MariamMohamedFawzy Done.

**Pull request merged**